static/js/settings_bots.js

```diff
@@ -282,6 +302,19 @@ export function set_up_bot_edit_form(bot) {
                     formData.append("bot_owner_id", human_user_id);
                 }
 
+                if (bot_type === OUTGOING_WEBHOOK_BOT_TYPE) {
+                    const service_payload_url = $("#edit_service_base_url").val();
+                    const service_interface = $("#edit_service_interface :selected").val();
+                    formData.append("service_payload_url", JSON.stringify(service_payload_url));
+                    formData.append("service_interface", service_interface);
+                } else if (bot_type === EMBEDDED_BOT_TYPE && service !== undefined) {
+                    const config_data = {};
+                    $("#config_edit_inputbox input").each(function () {
+                        config_data[$(this).attr("name")] = $(this).val();
+                    });
+                    formData.append("config_data", JSON.stringify(config_data));
```

Comment:
I am assuming we can allow admins to edit these configurations of bots not owned by them. Mentioning this since we did not discuss it before.

Reply:
Hmm, the `config_data` could contain secrets (e.g. an API key for a third-party service), so we might want to go with the same approach that we have for API keys, which IIRC is that administrators need to take control of the bot explicitly by assigning themselves as the owner to access it? But maybe worth starting a #backend thread if I'm confused.